firewalld.spec

```diff
@@ -119,15 +121,24 @@ firewalld.
 
 %prep
 %autosetup
+%if 0%{?rhel} < 8
+./autogen.sh --without-nft
```

Comment:
This change is not necessary as of 3f7419d. We only rely on python3-nftables and no longer check for the nft binary.